### PR TITLE
Custom MIR: Support `Rvalue::NullaryOp`

### PR DESCRIPTION
**Pull request by beepster4096**

### CODE REVIEW

**Comment by petrochenkov** [comment]
r? compiler

File: compiler/rustc_mir_build/src/build/custom/parse/instruction.rs [hunk 1]
```diff
@@ -158,6 +158,28 @@ impl<'tcx, 'body> ParseCtxt<'tcx, 'body> {
             },
             @call("mir_len", args) => Ok(Rvalue::Len(self.parse_place(args[0])?)),
             @call("mir_copy_for_deref", args) => Ok(Rvalue::CopyForDeref(self.parse_place(args[0])?)),
+            @call("mir_size_of", _args, substs) => {
```
[the remark this answers is not in the document]
Comment (oli-obk) [reply compiler/rustc_mir_build/src/build/custom/parse/instruction.rs:161]: Similar question for `args`, tho that is preexisting

**Comment by Dylan-DPC** [comment]
@DrMeepster any updates on this?

**Comment by Dylan-DPC** [comment]
Closing this as inactive. Feel free to reöpen this pr or create a new pr if you get the time to work on this. Thanks

**Pull request closed without merging**